src/EditorFeatures/Test2/IntelliSense/QuickInfoControllerTests.vb

```diff
@@ -22,24 +24,25 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             TestWorkspace.ResetThreadAffinity()
         End Sub
 
-        <WpfFact>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)>
         Public Sub InvokeQuickInfoWithoutDocumentShouldNotQueryProviders()
             Dim emptyProvider = New Mock(Of IDocumentProvider)
             emptyProvider.Setup(Function(p) p.GetDocumentAsync(It.IsAny(Of ITextSnapshot), It.IsAny(Of CancellationToken))).Returns(Task.FromResult(DirectCast(Nothing, Document)))
             Dim controller As Controller = CreateController(documentProvider:=emptyProvider, triggerQuickInfo:=True)
             controller.WaitForController()
 
-            GetMocks(controller).Provider.Verify(Function(p) p.GetItemAsync(It.IsAny(Of Document), It.IsAny(Of Integer), It.IsAny(Of CancellationToken)), Times.Never)
+            'GetMocks(controller).Provider.Verify(Function(p) p.GetItemAsync(It.IsAny(Of Document), It.IsAny(Of Integer), It.IsAny(Of CancellationToken)), Times.Never)
```

Comment:
Suspiciously commented code is suspicious. It does look dead, I think?

Reply:
yeah, it looks dead. This is from Matt's original change. I just didn't notice it. Thanks!

Reply:
fixed